tools/integration_tests/log_rotation/logrotate_logfile_test.go

```diff
@@ -62,7 +63,12 @@ func runOperationsOnFileTillLogRotation(t *testing.T, wg *sync.WaitGroup, fileNa
 		// Break the loop when log file is rotated.
 		fi, err := operations.StatFile(logFilePath)
 		if err != nil {
-			t.Errorf("stat operation on file %s: %v", logFilePath, err)
+			t.Logf("stat operation on file %s failed: %v", logFilePath, err)
+			if !retryStatLogFile {
```

Comment:
The issue might come again in case when rotation will happen twice for a file. 
I think, we should add a check to catch when error will happen back to back (in consecutive loops).

Reply:
Discussed offline that this is only for one log rotation operation, so this check won't be required.